engine.py

```diff
@@ -473,6 +473,9 @@ def save(self, document):
         """
         Save the document in place
         """
+        if not document.saved:
```

Comment:
Based on your [previous comment](https://github.com/shotgunsoftware/tk-photoshopcc/pull/88/files#r1116259219) on #88, this line should be:

```suggestion
        if document.saved:
```